### PR TITLE
MULTISITE-21923: [TK3] Refactor build-subsite-dist to allow the full codebase to be built.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by jonhy81** [comment]
Postponed for September.

**Comment by jonhy81** [comment]
Postponed for September.

File: includes/phing/build/main/subsite.xml [hunk 1]
```diff
@@ -32,6 +32,16 @@
             subsite-resources-copy,
             package-version-set,
             project-subsite-composer-no-dev">
+        <if>
```
Comment (jonhy81) [inline includes/phing/build/main/subsite.xml]: This should be include in a new target `build-dist` to make sure we don't get conflicts with projects overriding `build-subsite-dist`

File: includes/phing/build/main/subsite.xml [hunk 2]
```diff
@@ -21,19 +21,31 @@
...
+      subsite-make,
+      subsite-type-dist,
+      subsite-resources-copy,
+      package-version-set,
+      project-subsite-composer-no-dev">
+    <echo msg="Playbook for build-subsite-dist ended." />
+  </target>
+  
+  <!-- Port hardcoded section of pipeline into toolkit target to enable override. -->
+  <target  name="cerate-distribution" description="Create a package to be released in a docker image.">
```
Comment (ghost) [inline includes/phing/build/main/subsite.xml]: Typo in target name and one space too much.

**Review by jonhy81 — APPROVED** [review]

**Pull request merged**